src/test/java/com/amazon/janusgraph/ScenarioTests.java

```diff
@@ -90,6 +91,16 @@ public void lockingTest() throws BackendException, InterruptedException {
         createSchemaAndDemoLockExpiry(USE_STORAGE_NATIVE_LOCKING, USE_EDGESTORE_LOCKING, USE_GRAPHINDEX_LOCKING, 100);
     }
 
+    /**
+     * https://stackoverflow.com/questions/44535054/generalizing-dynamodb-janusgraph-factory-lock-and-schema-problems
+     * @throws Exception
+     */
+    @Test
+    public void tripleIngestTest(StandardJanusGraph graph) throws Exception {
```

Comment:
perhaps we will discover that the issue is different than what we originally thought, so lets revisit the name of this test once all the code is in here.